src/AllergyIntolerance-3-0/Cert/_reference/AllergyIntolerance20/cio1-nl-core-relatedperson-ABRAHAMB-M-H-BRAMPASVEER.xml

```diff
@@ -59,35 +59,24 @@
             </extension>
         </given>
     </name>
-    <telecom>
-        <system value="phone">
-            <extension url="http://nictiz.nl/fhir/StructureDefinition/code-specification">
-                <valueCodeableConcept>
-                    <coding>
-                        <system value="http://hl7.org/fhir/v3/AddressUse"/>
-                        <code value="MC"/>
-                        <display value="Mobiel telefoonnummer"/>
-                    </coding>
-                </valueCodeableConcept>
-            </extension>
-        </system>
+	<telecom>
+	    <extension url="http://nictiz.nl/fhir/StructureDefinition/zib-ContactInformation-TelecomType">
+            <valueCodeableConcept>
+                <coding>
+                    <system value="http://hl7.org/fhir/v3/AddressUse"/>
+                    <code value="MC"/>
+                    <display value="Mobiel telefoonnummer"/>
+                </coding>
+            </valueCodeableConcept>
+        </extension>
+        <system value="phone"/>
         <value value="06-12345670"/>
         <use value="mobile"/>
```

Comment:
ext:TelecomType 'MC' met .system = 'phone' en .use = 'mobile' is geen toegestane combinatie volgens de mapping tabel in het profiel https://simplifier.net/nictizstu3-zib2017/nl-core-contactpoint

Reply:
Moet volgens mij 
| Mobile phone number undetermined private/business  |                  | phone   | mobile |
zijn, dus de extensie mag weg.

Reply:
Hoewel hier: https://informatiestandaarden.nictiz.nl/wiki/MedMij:V2020.01/kw-MM-AIB-FHIR#Contactpersoon_-_met_id:_.27AI_Contactpersoon_01.27_-_1.1

Duidelijk nog van een code 'MC' gesproken wordt...

Reply:
Gezien de mapping in nl-core-contactpoint zou het in de mapping naar FHIR inderdaad moeten worden zoals Jorn aangeeft. Dus de extensie weg.